app/controllers/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+require './config/environment'
+
+class UsersController < Sinatra::Base
+  include Helper
+
+  configure do
+    enable :sessions unless test?
+    set :public_folder, 'public'
+    set :views, 'app/views'
+    set :session_secret, "secret"
+  end
+
+
+  get '/signup' do
+    if is_logged_in
+      # erb :'/tweets/index'
+      redirect to '/tweets'
+    else
+      erb :'users/create_user', layout: false
+    end
+  end
+
+  post '/signup' do
+    @user = User.create(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
Great job using ActiveRecord model validations! The usual pattern for this is to combine `User.new` and `if @user.save?` instead of `User.create` and checking the ID.